Headers/AppKit/NSSpeechSynthesizer.h

```diff
@@ -151,8 +151,13 @@ extern NSString *NSSpeechDictionaryEntryPhonemes;
 - (NSString *) phonemesFromText: (NSString *)text;
 @end
 
-// Speech synthesizer delegate informal protocol...
```

Comment:
I don't object it, but I'll note that there is no reason to remove this comment.